docs/configuration/registration/zgw.rst

```diff
@@ -112,6 +112,9 @@ Configuration
    * **Objects API - Objecttype**: Fill in the default URL of the ProductAanvraag OBJECTTYPE in the Objecttypes API *For example* ``https://example.com/api/v1/objecttypes``
    * **Objects API - Objecttype version**: Fill in the default version of the OBJECTTYPE in the Objecttypes API *For example:* ``1``
 
+#. You can map a form variable with a ZAAK property (eigenshap) by clicking the related button. A modal will open where you have to choose the variable along with providing a valid property name.
```

Comment:
```suggestion
#. You can map a form variable with a Zaak property (eigenshap) by clicking the related button. A modal will open where you have to choose the variable along with providing a valid property name.
```

By "related button", do you mean this one?

![image](https://github.com/open-formulieren/open-forms/assets/65306057/2b5b7334-2c96-4f78-a864-3a8097d16e80)

If this is the case, might be worth explicitly referencing the button label